todo_app/models.py

```diff
@@ -0,0 +1,22 @@
+from dataclasses import dataclass
+from typing import Tuple
+
+import todo_app.constants as constants
+from todo_app.trello import TrelloList, TrelloListWithCards
+
+@dataclass(frozen=True)
+class ViewModel:
+    lists: Tuple[TrelloListWithCards, ...]
+
+    @property
+    def todo_list(self):
```

Comment:
These properties work nicely, but note that we never use them outside of the tests! I suspect you wanted to keep your custom flexible-list behaviour and still follow the exercise, but it might have made more sense to pick an equivalent behaviour to implement & test so that the tests are genuinely providing value - e.g. a property to wrap the underlying tuple, to sort the lists in some way that we could use, or just add an extra property for the "other" lists, and then use those to impose a standard ordering in your templates, e.g. Todo -> Doing -> Done -> <Anything else>

Reply:
D'oh. I'd meant to make use of them in `index.html` (so that it prints lists To Do, In Progress, Done, and then any additional lists), but I never implemented that. I've created issue #13 to cover this.